docs/Data_Portal/Users_Guide/scRNA.md

```diff
@@ -258,9 +262,12 @@ Here, a violin plot is displayed, showing the distribution of the gene of intere
 [![Violin plot](./images/scrna/image18-summary.png)](./images/scrna/image18-summary.png 'Click to see the full image.')
 
 Measures of central tendency such as mean, median, variance, and others are provided, along with a table showing pairwise statistical comparisons. This comparison is performed between all clusters for the particular gene of interest, with their respective p-values (generated by the Wilcoxon Rank Sum test method). Several options to refine the violin plot visualization include:
+1.	The ability to alter the orientation of the plot (horizontal or vertical).
```

Comment:
Add a return before this line.  I think the numbering will not render as a list without a return.

Reply:
I'm not sure if this matters, but the tab between the numbers and the steps might cause an issue.  I recommend changing them to spaces.